src/masternodes/rpc_proposals.cpp

```diff
@@ -9,64 +9,57 @@ struct VotingInfo {
+    auto proposalId        = propId.GetHex();
+    auto creationHeight    = static_cast<int32_t>(prop.creationHeight);
+    auto title             = prop.title;
+    auto context           = prop.context;
+    auto contextHash       = prop.contextHash;
+    auto type              = static_cast<CPropType>(prop.type);
+    auto typeString        = CPropTypeToString(type);
+    auto amountValue       = ValueFromAmount(prop.nAmount);
+    auto payoutAddress     = ScriptToString(prop.address);
+    auto currentCycle      = static_cast<int32_t>(prop.cycle);
+    auto totalCycles       = static_cast<int32_t>(prop.nCycles);
+    auto cycleEndHeight    = static_cast<int32_t>(prop.cycleEndHeight);
     auto proposalEndHeight = static_cast<int32_t>(prop.proposalEndHeight);
-    auto votingPeriod = static_cast<int32_t>(prop.votingPeriod);
-    bool isEmergency = prop.options & CPropOption::Emergency;
-    auto quorum = prop.quorum;
+    auto votingPeriod      = static_cast<int32_t>(prop.votingPeriod);
+    bool isEmergency       = prop.options & CPropOption::Emergency;
+    auto quorum            = prop.quorum;
     auto approvalThreshold = prop.approvalThreshold;
-    auto status = static_cast<CPropStatusType>(prop.status);
-    auto statusString = CPropStatusToString(status);
-    auto feeTotalValue =  ValueFromAmount(prop.fee);
-    auto feeBurnValue = ValueFromAmount(prop.feeBurnAmount);
+    auto status            = static_cast<CPropStatusType>(prop.status);
+    auto statusString      = CPropStatusToString(status);
+    auto feeTotalValue     = ValueFromAmount(prop.fee);
+    auto feeBurnValue      = ValueFromAmount(prop.feeBurnAmount);
 
-    auto quorumString = strprintf("%d.%02d%%", quorum / 100, quorum % 100);
+    auto quorumString            = strprintf("%d.%02d%%", quorum / 100, quorum % 100);
     auto approvalThresholdString = strprintf("%d.%02d%%", approvalThreshold / 100, approvalThreshold % 100);
 
-    auto votesPossible = -1;
-    auto votesPresent = -1;
-    auto votesPresentPct = -1;
-    auto votesYes = -1;
-    auto votesYesPct = -1;
+    auto votesPossible                = -1;
+    auto votesPresent                 = -1;
+    auto votesPresentPct              = -1;
+    auto votesYes                     = -1;
+    auto votesYesPct                  = -1;
     std::string votesPresentPctString = "-1";
-    std::string votesYesPctString = "-1";
+    std::string votesYesPctString     = "-1";
 
     auto isVotingInfoAvailable = votingInfo.has_value();
     if (isVotingInfoAvailable) {
-        votesPresent = votingInfo->votesPresent;
-        votesYes = votingInfo->votesYes;
+        votesPresent  = votingInfo->votesPresent;
+        votesYes      = votingInfo->votesYes;
         votesPossible = votingInfo->votesPossible;
 
         votesPresentPct = lround(votesPresent * 10000.f / votesPossible);
-        auto valid = votesPresentPct > quorum;
+        auto valid      = votesPresentPct > quorum;
```

Comment:
I would remove this valid too, and just check if votesPresent > 0 and always show `VotesYesPct` then!

Reply:
Removed the `valid` check.
`votesPresent` > 0 is already handled here https://github.com/DeFiCh/ain/blob/221a8489a632098f1dbb1cc1af84de98aba369c6/src/masternodes/rpc_proposals.cpp#L629